lib/wwtd/ruby.rb

```diff
@@ -24,7 +28,7 @@ def switch_statement(version, options={})
           end
         else
           if ruby_root = ENV["RUBY_ROOT"] # chruby or RUBY_ROOT set
-            switch_via_env(File.dirname(ruby_root), version)
+            switch_via_env(ruby_root, version)
```

Comment:
Not for chruby, I discovered it while trying to make the RUBY_ROOT option work. I think it's a bug.